### DISCUSSION TITLE
[vscode] Strange notification

### DISCUSSION THREAD

**Discussion by angelhdzmultimedia** [vuejs/language-tools#4381] [Q&A] [answered]
Lately I'm getting this notification:
![image](https://github.com/vuejs/language-tools/assets/10191085/5264790a-ede3-4db1-bfbe-4a176e021dd2)

**Comment by so1ve** [comment, marked as answer]
Literally it means runem.lit-plugin doesn't work with Vue - Official. So we use a language server instead of a typescript plugin.
